routers/repo/topic.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // TopicPost response for creating repository
```

Comment:
The lint error is right here `TopicPost` should be `TopicsPost` to match name of function

Reply:
fixed